tests/test_modeling_roberta.py

```diff
@@ -198,6 +198,61 @@ def create_and_check_for_causal_lm(
         result = model(input_ids, attention_mask=input_mask, token_type_ids=token_type_ids, labels=token_labels)
         self.parent.assertEqual(result.logits.shape, (self.batch_size, self.seq_length, self.vocab_size))
 
+    def create_and_check_decoder_model_past_large_inputs(
+        self,
+        config,
+        input_ids,
+        token_type_ids,
+        input_mask,
+        sequence_labels,
+        token_labels,
+        choice_labels,
+        encoder_hidden_states,
+        encoder_attention_mask,
+    ):
```

Comment:
This test sometimes fails, not sure why. Manually checked the generated output of few models Roberta2Roberta models with `use_cahce=True` and it matches the output without caching

Reply:
Hmm, guess this needs a bit more investigation then. This test should pass with a tolerance of 1e-3 -> there might be a subtlety that's different - usually if the differences are so small it's tied to some errors regarding the `encoder_attention_mask` or cached encoder hidden_states